components/Input/input.css

```diff
@@ -0,0 +1,54 @@
+.input-group {
+  display: flex;
+  flex-direction: column;
+  margin: 1rem 0;
+}
+
+.input-group label {
+  font-size: var(--text-sm);
+  margin-bottom: 4px;
+  font-weight: var(--font-medium);
+}
+
+.input {
+  font-family: "Inter", sans-serif;
```

Comment:
Try to use font family as a variable don't hard code

Reply:
Done ty!